deps/jscshim/webkit.gypi

```diff
@@ -11,7 +11,8 @@
                                     '<!(pwd)/webkit/WebKitBuild/lib/<(jsc_build_config)/<(STATIC_LIB_PREFIX)JavaScriptCore<(STATIC_LIB_SUFFIX)'],
       }, {
         'webkit_output_dir': './webkit/WebKitBuild/<(jsc_build_config)',
-        'webkit_output_libraries': ['<!(python -c "import os; print os.getcwd()")/webkit/WebKitBuild/<(jsc_build_config)/lib/<(STATIC_LIB_PREFIX)WTF<(STATIC_LIB_SUFFIX)',
+        'webkit_output_libraries': ['<!(python -c "import os; print os.getcwd()")/webkit/WebKitBuild/<(jsc_build_config)/lib/<(STATIC_LIB_PREFIX)bmalloc<(STATIC_LIB_SUFFIX)',
+                                    '<!(python -c "import os; print os.getcwd()")/webkit/WebKitBuild/<(jsc_build_config)/lib/<(STATIC_LIB_PREFIX)WTF<(STATIC_LIB_SUFFIX)',
```

Comment:
Ad-hoc change. This breaks Windows port. In Linux, bmalloc is enabled. So we should have `libbmalloc.a`.